build.gradle.kts

```diff
@@ -1,9 +1,49 @@
+import net.pwall.json.kotlin.codegen.gradle.JSONSchemaCodegenPlugin
+import net.pwall.json.kotlin.codegen.gradle.JSONSchemaCodegenTask
+
+buildscript {
```

Comment:
See comments below. I would like to keep the root project file small. let's move this into a `csaf-schema` submodule